src/main/java/io/github/thebusybiscuit/slimefun4/implementation/tasks/TickerTask.java

```diff
@@ -300,12 +300,13 @@ private void tick() {
 
     @ParametersAreNonnullByDefault
     private void tickChunk(ChunkPosition chunk, Collection<BlockTicker> tickers, Collection<Location> locations) {
+        // Only continue if the Chunk is actually loaded
+        if (!chunk.isLoaded()) {
+            return;
+        }
         try {
-            // Only continue if the Chunk is actually loaded
-            if (chunk.isLoaded()) {
-                for (Location l : locations) {
-                    tickLocation(tickers, l);
-                }
+            for (Location l : locations) {
+                tickLocation(tickers, l);
             }
         } catch (ArrayIndexOutOfBoundsException | NumberFormatException x) {
             SlimefunPlugin.logger().log(Level.SEVERE, x, () -> "An Exception has occurred while trying to resolve Chunk: " + chunk);
```

Comment:
this try-catch still seems to be redundant and just unnecessary overhead. Did you find a reason to keep it?

Reply:
There is no reason to keep it, see my comment.

Reply:
As mentioned above, I'm leaving that to be done by Biscuit in #2996